packages/forms-web-app/src/dynamic-forms/has-questionnaire/questions.js

```diff
@@ -95,6 +95,12 @@ exports.questions = {
 		fieldName: 'letters-to-neighbours',
 		validators: [new RequiredValidator()]
 	}),
+	uploadLettersToNeighbours: new MultiFileUploadQuestion({
+		title: 'Uploaded letters',
+		question: 'Upload the letters and emails',
+		fieldName: 'upload-letters-emailss',
+		validators: [new RequiredValidator('You must add your documents')]
```

Comment:
for multifile validation, this line should now read:
`validators: [new RequiredFileUploadValidator(), new MultifileUploadValidator()]`

Reply:
(you won't need to specify the error message as it is the default for the RequiredFileUploadValidator)